organization/_posts/2023/2023-07-06-coordination.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "HSF Coordination Meeting #253, 6 July 2023"
+layout: plain_toc
+---
+
+## Attending
+
+Present/Contributing: Graeme Stewart, Benedikt Hegner, Eduardo Rodrigues, Claire Antel, Patrick Gartung, Michel Jouvin, Pere Mato, Liz Sexton-Kennedy, Nick Smith, Patick Gartung, Sudhir Malik, Tamas Gal, Valentin Volkl, Markus Diefenthaler, Matthew Feickert, Wouter Deconinck, Giulia Casarosa
+
+Apologies/Contributing: Caterina Doglioni, Efe Yazgan
+
+## News, general matters, announcements
+
+### Google Summer of Code, 2023
+
+- Representative for the mentors summit to be decided.
+  - It shouldn't just be one of the coordinators, should be someone more involved in mentoring.
+
+### LHCC
+
+#### LHCC 154 (June)
+
+The [LHCC Minutes](https://cds.cern.ch/record/2861565?ln=en) are now published. On analysis facilities:
+
+> A focus session on Analysis Facilities has been organised during the LHCC week, where the current R&D work happening within the community and discussed in the HSF Analysis Facilities Forum has been presented. The HSF Analysis Facilities Forum has focused on the technical building blocks that can make the user experience more productive. Novel concepts of Analysis Facilities, integrating interactivity, scalability (offload) and Machine Learning tools are being built.
+
+> The **LHCC recommends** that experiments engage in the process of developing and defining the structure of the future Analysis Facilities and **requests** they produce a document which defines the use cases in order to establish realistic benchmarks. This process should be coordinated with the HL-LHC Computing and SW review panel. The document is expected to be regularly updated in the process towards the HL-LHC.
```

Comment:
```suggestion
> The **LHCC recommends** that experiments engage in the process of developing and defining the structure of the future Analysis Facilities and **requests** they produce a document that defines the use cases in order to establish realistic benchmarks. This process should be coordinated with the HL-LHC Computing and SW review panel. The document is expected to be regularly updated in the process towards the HL-LHC.
```

Reply:
It's verbatim from the minutes, so I don't want to change that, even if the English could be improved!